integration-cli/docker_cli_build_test.go

```diff
@@ -863,10 +862,10 @@ func (s *DockerSuite) TestBuildCopyWildcard(c *check.C) {
 	ctx, err := fakeContext(fmt.Sprintf(`FROM busybox
 	COPY file*.txt /tmp/
 	RUN ls /tmp/file1.txt /tmp/file2.txt
-	RUN mkdir /tmp1
+	RUN sh -c "mkdir /tmp1"
```

Comment:
an a few unrelated questions... does the busybox for Windows have "sh" ?  And does the busybox for windows support / and \  as dir separators ?